src/accelerate/data_loader.py

```diff
@@ -633,6 +635,7 @@ def prepare_data_loader(
     rng_types: Optional[List[Union[str, RNGType]]] = None,
     dispatch_batches: Optional[bool] = None,
     even_batches: bool = True,
+    slice_fn: Optional[Callable] = None,
```

Comment:
The name should make it clear this is only for `dispatch` mode. This also needs to be properly documented.

Reply:
shall I rename to `slice_fn_for_dispatch`? I will add it to the docstring.

Reply:
Yes, that's good for here! Thanks!